File: test/fixtures/matlab/post_with_single_quotes_inside_double_quotes.m
%% Web Access using Data Import and Export API
uri = 'http://localhost:28139/';
body = 'foo=''bar''';
options = weboptions('MediaType', 'application/x-www-form-urlencoded');
response = webwrite(uri, body, options);

%% HTTP Interface
import matlab.net.*
import matlab.net.http.*
import matlab.net.http.io.*

header = HeaderField('Content-Type', 'application/x-www-form-urlencoded');
uri = URI('http://localhost:28139/');
body = FormProvider('foo', '''bar''');
response = RequestMessage('post', header, body).send(uri.EncodedURI);

File: test/fixtures/node-request/post_with_single_quotes_inside_double_quotes.js
var request = require('request');

var headers = {
    'Content-Type': 'application/x-www-form-urlencoded'
};

var dataString = "foo='bar'";

var options = {
    url: 'http://localhost:28139/',
    method: 'POST',
    headers: headers,
    body: dataString
};

function callback(error, response, body) {
    if (!error && response.statusCode == 200) {
        console.log(body);
    }
}

request(options, callback);
